### PR TITLE
feat(i18n): add Belarusian translation

### PR DESCRIPTION
**Pull request by gleb-kun**
Added translation into Belarusian language.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 public
 resources
 assets/jsconfig.json
-.hugo_build.lock
\ No newline at end of file
+.hugo_build.lock
```
Comment (CaiJimmy) [inline .gitignore:4]: Can you please remove the changes made to `.gitignore`?

**Review by CaiJimmy — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: .gitignore (hunk 1 above)
Comment (gleb-kun) [reply .gitignore]: I did it.

Comment (gleb-kun) [reply .gitignore]: Should I have left the exception for temporary folder `.idea`?

**Comment by ottok** [comment]
@CaiJimmy This seems ready, can you do final review and consider merging?

File: .gitignore (hunk 1 above)
Comment (CaiJimmy) [reply .gitignore]: > Should I have left the exception for temporary folder `.idea`?

No, it's fine like this. Thank you!

**Review by CaiJimmy — APPROVED** [review]
Thank you!

File: .gitignore (hunk 1 above)
Comment (gleb-kun) [reply .gitignore]: You're welcome.

**Pull request merged**